data/modules/BreakdownServicing/BreakdownServicing.lua

```diff
@@ -248,28 +248,37 @@ local onEnterSystem = function (ship)
 	else
 		return -- Don't care about NPC ships
 	end
+
+	-- The more engineers the more likely to be saved (rule doesn't apply to cooking):
 	local saved_by_this_guy = savedByCrew(ship)
```

Comment:
The only line unchanged, and, bizarrely, the one I dislike the most. This name is crap! Is it a bool, is it a table/object?

Reply:
I agree, I had to read Character.lua to see what it was. Not to mention: IT'S SEXIST!!!